File: codewars/kyu_6/php/multiples_of_3_or_5.php
<?php

# https://www.codewars.com/kata/multiples-of-3-or-5/

function solution($number){
    $result = array();
    for ($i = 0; $i < $number; $i++) {
        if (($i % 3 == 0) || ($i % 5) == 0) {
            array_push($result, $i);
        }
    }
    return array_sum($result);
}

?>

File: codewars/kyu_6/python/multiples_of_3_or_5.py
# https://www.codewars.com/kata/multiples-of-3-or-5/

def solution(number):
    result = []
    for x in range(number):
        if x % 3 == 0 or x % 5 == 0:
            result.append(x)
    return sum(result)
